how-to/how-to-use-virtio-fs-with-kata.md

```diff
@@ -22,30 +22,39 @@ This document describes how to get Kata Containers to work with virtio-fs.
 ## Pre-requisites
 
 * Before Kata 1.8 this feature required the host to have hugepages support enabled. Enable this with the `sysctl vm.nr_hugepages=1024` command on the host.
+  In later versions of Kata, virtio-fs leverages `/dev/shm` as the shared memory backend. The default size of `/dev/shm` on a system is typically half of the 
+  total system memory. This can pose a physical limit to the maximum number of pods that can be launched with virtio-fs. This can be overcome by increasing the size of `/dev/shm`.
```

Comment:
nit:
```suggestion
  total system memory. This can pose a limit to the maximum number of pods that can be launched with virtio-fs. This can be overcome by increasing the size of `/dev/shm`.
```

can you describe how to increase this?

Reply:
@egernst Fixed